### PR TITLE
Add a random payload with the given size to requests

### PR DESCRIPTION
**Pull request by whalebot-helmsman**
Continuation of work from https://github.com/scrapy/scrapy-bench/pull/36

### CODE REVIEW

File: scrapy_bench/middlewares.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def handle_request(self, smth):
+        if not isinstance(smth, Request):
+            return smth
+
+        request = smth
+
+        if self.size == 0:
+            return request
+
+        request.meta['random_payload'] = os.urandom(self.size)
```
Comment (lopuhin) [inline scrapy_bench/middlewares.py:30]: @whalebot-helmsman what would be the consequence of settings this meta field, how does this affect the benchmark? It won't be sent over the network, right?

Comment (lopuhin) [reply scrapy_bench/middlewares.py:30]: Sorry, I didn't check #36 - now I see the context, although still not 100% clearly. Could you please document what is the goal of this setting somewhere, in the README or in the code?

Comment (whalebot-helmsman) [reply scrapy_bench/middlewares.py:31]: done in https://github.com/scrapy/scrapy-bench/pull/37/commits/bae275e47049362a617785d3369b94207396e1c1

Comment (lopuhin) [reply scrapy_bench/middlewares.py:31]: Thanks @whalebot-helmsman , makes sense - so this meta field would be serialized with the request when using a disk queue, this is why it matters?

**Review by lopuhin — APPROVED** [review]
Looks great, thanks @whalebot-helmsman 👍

**Pull request merged**